src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -830,6 +835,54 @@ public function isAbstract()
     }
 
     /**
+     * Whether this definition is deprecated, that means it should not be called
+     * anymore.
+     *
+     * @param bool   $boolean
+     * @param string $template Template message to use if the definition is deprecated
+     *
+     * @return Definition the current instance
+     *
+     * @api
+     */
+    public function setDeprecated($boolean = true, $template = 'The service %service% is deprecated. You should stop using it, as it will soon be removed.')
```

Comment:
In order to remove the duplication of the default message, I'd put `$template = null` here